deployment/CP05MOAS-GL336_Deploy.csv

```diff
@@ -11,45 +11,45 @@ CT20160808,,TS20160818,,CP05MOAS-GL336-02-FLORTM000,2,1,2016-08-09T13:54:00,2016
+CT20180724,,AR31,,CP05MOAS-GL336-04-DOSTAM000,7,1,2018-07-24T14:33:00,2018-10-25T23:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00192,39.94883,-70.67285,,1000,1000,FZ-1
+CT20180724,,AR31,,CP05MOAS-GL336-05-PARADM000,7,1,2018-07-24T14:33:00,2018-10-25T23:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50204,39.94883,-70.67285,,1000,1000,FZ-1
+AR34,,AR34,,CP05MOAS-GL336-00-ENG000000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,39.95807,-71.00037,,1000,1000,FZ-2
+AR34,,AR34,,CP05MOAS-GL336-01-ADCPAM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-43111,39.95807,-71.00037,,1000,1000,FZ-2
+AR34,,AR34,,CP05MOAS-GL336-02-FLORTM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-03206,39.95807,-71.00037,,1000,1000,FZ-2
+AR34,,AR34,,CP05MOAS-GL336-03-CTDGVM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09060,39.95807,-71.00037,,1000,1000,FZ-2
+AR34,,AR34,,CP05MOAS-GL336-04-DOSTAM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,39.95807,-71.00037,,1000,1000,FZ-2
+AR34,,AR34,,CP05MOAS-GL336-05-PARADM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,39.95807,-71.00037,,1000,1000,FZ-2
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-00-ENG000000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-01-ADCPAM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-43111,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-02-FLORTM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-03206,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-03-CTDGVM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09060,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-04-DOSTAM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
-TS20190516,,UNRECOVERED,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
\ No newline at end of file
+TS20190516,,UNRECOVERED,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea

```

Comment:
looks good - weird highlighted space at end of the deployment 4 lines but I think it's ok.